manimlib/camera/camera.py

```diff
@@ -338,15 +338,15 @@ def set_cairo_context_path(self, ctx, vmobject):
             return
 
         ctx.new_path()
-        subpaths = vmobject.get_subpaths_from_points(points)
+        subpaths = vmobject.gen_subpaths_from_points_2d(points)
```

Comment:
Wouldn't this give the incorrect answers for 3D scenes? I think you'll have to check whether or not this is being used with a 3D scene or not.

Reply:
We're in a 2D world at this point in `set_cairo_context_path()` and the Z value is just dead weight. Just above this call we have called `self.transform_points_pre_display()` to get the points, which the 3D camera uses to project objects onto 2D. Note that afterwards in this function the only use of `subpaths` is to eventually call `ctx.curve_to(*p1[:2], *p2[:2], *p3[:2])`, which only takes x,y coordinates, and the Z value is dropped from each point.